src/main/java/org/postgresql/util/PGobject.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Taken from Postgres JDBC Driver 9.4 (BSD license terms)
+ * 
+ * License : https://jdbc.postgresql.org/about/license.html
```

Comment:
@frode-carlsen You can't take this package namespace. See XAConnection for an example, and maintain license and copyright as Dave states